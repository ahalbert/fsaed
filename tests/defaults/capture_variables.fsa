/beep/ ->
/boop/ {capture $_ myvar = $_ ->} 
/buzz/ {print myvar}
