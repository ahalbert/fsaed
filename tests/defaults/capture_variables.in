beep
boop
foo
bar
baz
buzz
