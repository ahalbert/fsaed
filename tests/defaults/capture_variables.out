beep
foo
bar
baz
boop
buzz
